### DISCUSSION TITLE
Deploying Next.js app in a Web Server (Apache)

### DISCUSSION THREAD

**Discussion by hgf777-br** [vercel/next.js#22190] [Help]
Hello to all. 
I am new with Next but have been using react for some time. I want to know how to deploy a Next app like a I do with a React one. 
With the "create-react-app" I can build a version of it and deploy to a folder in my server and  when I enter that path with the browser I got my react app (this works fine for me).
But when I try with Next I can´t get it working right. For static pages it works fine with the "export" command. But with the build, for dynamic pages, I am having some trouble.
How should I do this deploy?

**Comment by MarwenLabidi** [comment]
hello i have the same issue 
did you find a solution for that ?

**Comment by DenisBessa** [comment]
Have you considered deploying it using docker?

It's pretty simple and you don't have to deal with Apache or any other internal server stuff.

Take a look at this guide: https://nextjs.org/docs/pages/building-your-application/deploying#docker-image

**Reply by MarwenLabidi** [reply]
thank you man
